modules/programs/neovim.nix

```diff
@@ -351,10 +342,10 @@ in {
     programs.neovim.generatedConfigViml = neovimConfig.neovimRcContent;
 
     programs.neovim.generatedConfigs = let
-      grouped = lib.lists.groupBy (x: x.type) pluginsNormalized;
+      grouped = builtins.groupBy (x: x.type) pluginsNormalized;
```

Comment:
lib.lists allows to target older nix as well so should be better to keep it.

Reply:
Ok. The one in `builtins` is a lot more efficient though, so it would be good to switch to it at some point.

Reply:
it is the one used if available see https://github.com/NixOS/nixpkgs/blob/2671a557503f7462a5e9147ba728caf9dce4bff4/lib/lists.nix#L346